pkg/controller/resourcequota/resource_quota_controller_test.go

```diff
@@ -136,6 +136,66 @@ func newTestPods() []runtime.Object {
 	}
 }
 
+func newBestEffortTestPods() []runtime.Object {
+	return []runtime.Object{
+		&v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{Name: "pod-running", Namespace: "testing"},
+			Status:     v1.PodStatus{Phase: v1.PodRunning},
+			Spec: v1.PodSpec{
+				Volumes:    []v1.Volume{{Name: "vol"}},
+				Containers: []v1.Container{{Name: "ctr", Image: "image", Resources: getResourceRequirements(getResourceList("", ""), getResourceList("", ""))}},
+			},
+		},
+		&v1.Pod{
+			ObjectMeta: metav1.ObjectMeta{Name: "pod-running-2", Namespace: "testing"},
```

Comment:
i'm not seeing the difference between this test case and the previous one.

Reply:
@sjenning  Assuming "this test case and the previous one" refering to `newBestEffortTestPods()` and `newTestPods()`, this utility function has two best effort pods while in previous one there is no best effort pods. This utility fucntion is added to test `ResourceQuotaScopeBestEffort`.

Reply:
yeah, nevermind